test/Microsoft.Health.Fhir.Shared.Tests.Integration/Features/ChangeFeed/SqlServerFhirResourceChangeCaptureDisabledTests.cs

```diff
@@ -40,7 +40,7 @@ public async Task GivenADatabaseSupportsResourceChangeCapture_WhenResourceChange
             FhirStorageTestsFixture fhirStorageTestsFixture = null;
             try
             {
-                string databaseName = $"FHIRRESOURCECHANGEDISABLEDTEST_{DateTimeOffset.UtcNow.ToUnixTimeSeconds()}_{BigInteger.Abs(new BigInteger(Guid.NewGuid().ToByteArray()))}";
+                string databaseName = SqlServerFhirStorageTestsFixture.GetDatabaseName($"ChangeCaptureDisabled");
```

Comment:
nit: no longer string interpolation
```suggestion
                string databaseName = SqlServerFhirStorageTestsFixture.GetDatabaseName("ChangeCaptureDisabled");
```

Reply:
:-)